### PR TITLE
chore(EMS-1491): Rename "exporter" to "account"

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR renames an "exporter" to an "account". This makes the data/codebase more scalable because in the future, the user/applicant of an application could _not_ be an exporter and instead could be a broker for example. 

## Changes
- Rename DB/API schema field, `Exporter`, to `Account`.
- Rename the DB/API's Application and Exporter/Account relationship to be called `owner` instead of `exporter`.
- Rename all `exporter` related variables to `account`, `exporters` to `accounts`, `exporterEmail` to `accountEmail`, `exporterId` to `accountId`, etc.
- Rename all "exporter" related files and functions to "account" throughout the API, UI, GQL, E2E suite.

### CODE REVIEW

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Looks good, one minor comment to address please.

File: e2e-tests/cypress/support/insurance/account/get-account-by-email.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * getAccountByEmail
+ * Get an exporter directly from the API,
+ * @param {String} Account email
+ * @returns {Object} Account
+ */
+const getAccountByEmail = (email) => {
+  try {
+    api.getAccountByEmail(email).then((response) => {
+      if (!response.body || !response.body.data) {
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply e2e-tests/cypress/support/insurance/account/get-account-by-email.js:12]: Thank you, good shout, have added this in.

File: src/api/test-mocks/index.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const mockExporterCompany = {
 
 export const mockBuyer = {
   companyOrOrganisationName: 'Mock buyer',
+  // buyerLocation: mockCountries[0].name,
```
Comment (Zainzzkk) [inline src/api/test-mocks/index.ts:39]: remove?

Comment (ttbarnes) [reply src/api/test-mocks/index.ts:39]: Great spot! Since everything is green can I do this in the next PR? Removed it locally already

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**